app/lib/common/themes/components/input_decoration_theme.dart

```diff
@@ -11,8 +11,8 @@ var inputDecorationTheme = InputDecorationTheme(
     borderRadius: BorderRadius.circular(12),
   ),
   enabledBorder: OutlineInputBorder(
-    borderSide: BorderSide(
-      color: brandColor,
+    borderSide: const BorderSide(
+      color: Colors.white30,
```

Comment:
I thought the idea was to reduce colors (and make them more semantic), now I am reading Colors.white30 and white12 which sounds like hard coding specific material colors that would not work well if we switched the surface color to e.g. light gray... Or maybe I misunderstood the goal?